packages/viewer/src/commands/streamCommands.ts

```diff
@@ -47,7 +49,133 @@ export function startStream(
   };
 }
 
+export function createSceneAlteration(
+  sceneViewId: UUID.UUID,
+  query: QueryExpression,
+  operations: OperationDefinition[]
+): Command<Promise<vertexvis.protobuf.stream.IStreamResponse>> {
+  return ({ stream }: CommandContext) => {
+    const pbOperations: vertexvis.protobuf.stream.ISceneOperation[] = [
+      buildSceneOperation(query, operations),
+    ];
+    const request: vertexvis.protobuf.stream.ICreateSceneAlterationRequest = {
+      sceneViewId: new vertexvis.protobuf.core.Uuid({
+        hex: sceneViewId,
+      }),
+      operations: pbOperations,
+    };
+
+    return stream.createSceneAlteration(request);
+  };
+}
+
+function buildSceneOperation(
```

Comment:
This adds a lot of new behavior. I'd recommend moving `buildSceneOperation` and the functions it calls into a new module.